Library/Homebrew/cmd/gist-logs.rb

```diff
@@ -40,6 +47,19 @@ def gistify_logs(f)
     puts url if url
   end
 
+  def brief_build_info(f)
+    build_time_str = f.logs.ctime.strftime("%Y-%m-%d %H:%M:%S")
+    hostname = Socket.gethostname
```

Comment:
Moving it inside the `if`. It was just leftover here from when I originally made it unconditional.

I think we really ought to have an option or environment variable here. It's not huge, but exposing the hostname is potentially personally identifying information, and for bug reports which may have security implications, an attack surface. These gists are public. And `gist-logs` has been around for a while and it hasn't exposed info like this in the past. So I think we should make it explicitly opt-on.

Conversely, it is not a big deal to leave it out IMHO: including the host name is only useful for users who run Homebrew on several different machines and need to distinguish between them in their bug reports. (e.g. I have hosts like `foo-yos`, `foo-elcap`, `foo-no-CLI` for different configurations that are quickly identifiable by name.) That sounds like an advanced use case.